src/main/java/aquality/appium/mobile/elements/ScreenSizeType.java

```diff
@@ -0,0 +1,6 @@
+package aquality.appium.mobile.elements;
+
+public enum ScreenSizeType {
```

Comment:
is it about screen orientation (landscape and portrait)? if so, can we rename appropriate to this?

Reply:
@DmitryBogatko, not really. It's referring to Mobile device Screen Size (not to be confused with Display Size), which is usually described by its Width and Length. But this enum is no longer necessary as the swipe method will take Points as arguments.